### DISCUSSION TITLE
JPA에서 다대다 중간 테이블의 양쪽에 cascade를 걸어두면 위험한 점

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#151] [1. 개발 관련 글 모음]
https://velog.io/@shkisme/JPA%EC%97%90%EC%84%9C-%EC%96%91%EC%AA%BD%EC%97%90-cascade%EB%A5%BC-%EA%B1%B8%EB%A9%B4-%EB%B0%9C%EC%83%9D%ED%95%98%EB%8A%94-%EB%AC%B8%EC%A0%9C

이건 제가 겨울방학때 진짜 삽질 오래 했던 문제였었습니다... 이걸 이제야 정리를 해보네요.
그때 정리를 안해두니까 이번에도 뭐가 문제였는지 기억도 안나고 좀 해맸습니다...ㅎㅎ

키글챌 최고👍

다음 글에서는 진짜로 orphanRemoval 사용에 따른 성능 문제를 다뤄보겠습니다~

**Comment by 02ggang9** [comment]
글 정말 잘 봤습니다~~ 글 내용대로 두 군데 다 걸어버리면 위험하네요.. 잘 배웠습니다~!!

**Reply by shkisme** [reply]
읽어주셔서 감사합니당 😇
